bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/WprimeToTauNu_LO/customize_cards.py

```diff
@@ -0,0 +1,42 @@
+import fileinput
+import sys 
+import os
+import shutil
+from WprimeWidth import *
+
+#### Set mass ranges to produce ####
+masses = range(600, 6800, 200)
+
+#### Set couplings to produce ####
+kR = [0.0001, 0.001, 0.01, 0.1, 1.0, 2.0, 3.0, 5.0]
+
+#### Set dir with template cards to adjust ####
+template_dir = "WprimeToTauNu_M-6000_kR-0.01_LO_NNPDF31nnlo/"
+#### Loop through all masses ####
+for mass in masses:
+#### Loop through all couplings ####
+    for coupling in kR: 
+        print(WprimeWidth(mass,coupling))
+        directory = "WprimeToTauNu_M-"+str(mass)+"_kR-"+str(coupling)+"_LO_NNPDF31nnlo" # Set the output name
+        if os.path.exists(directory): # Check if directiory exists already
+            shutil.rmtree(directory)  # Delete if it already exists
```

Comment:
These lines would remove all files stored in your template directory when reaching the 6000 GeV mass point and kR = 0.01

Reply:
Yes, sorry, locally I used a template called in another way. That is why there was no conflict. They are unnecessary for this case and can be deleted.

Reply:
I removed these lines from the code. Thank you for noticing @mlizzo .